Mods/CombatExtended/Languages/Russian/DefInjected/ApparelLayerDef/ApparelLayerDefs.xml

```diff
@@ -1,17 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <LanguageData>
 
-  <!-- EN: webbing -->
-  <Webbing.label>тесьма</Webbing.label>
-  <!-- EN: backpack -->
-  <Backpack.label>рюкзак</Backpack.label>
-  <!-- EN: shield -->
-  <Shield.label>щит</Shield.label>
-  <!-- EN: skin (head) -->
-  <OnHead.label>на голове</OnHead.label>
-  <!-- EN: strapped (head) -->
-  <StrappedHead.label>привязанное к голове</StrappedHead.label>
-  <!-- EN: middle (head) -->
-  <MiddleHead.label>на голове</MiddleHead.label>
```

Comment:
You sure we don't have this layer anymore? I am not certain, but it may be something we added ourselves

Reply:
At least the game didn't complain about missing cross-reference when loading, I'm not sure if other addon added them, other than More Content, Vile's mods

Reply:
If your refering to MiddleHead, yeah i dont think its used anymore